packages/gatsby-plugin-offline/README.md

```diff
@@ -197,8 +197,16 @@ Gatsby offers great SEO capabilities and that is no different with `gatsby-plugi
 
 To see the HTML data that crawlers will receive, run this in your terminal:
 
-```bash
-curl https://www.yourdomain.tld
+**on Windows (using powershell):**
```

Comment:
This seems to remove the previous bash instructions, just want to make sure that was intentional.

Reply:
I have both now :) Curl isn't super common in the Windows world so I've added a Linux and windows command.